### DISCUSSION TITLE
Where is the retroarch configuration file located?

### DISCUSSION THREAD

**Discussion by realjackwang** [OnionUI/Onion#518] [Q&A] [answered]
Onion wiki says the config file located in Saves/config, but when I open retroarch and press save config, the on-screen notification says the config file saved at ./.retroarch/retroarch.cfg, which really confuse me.

Do I need to backup the config file in  ./.retroarch/retroarch.cfg too when I upgrade Onion? and where is the truely retroarch config file located?

**Comment by Sichroteph** [comment, marked as answer]
General settings are saved in .RetroArch 
Emulator settings (filters, key bindings etc) are saved in Saves

**Reply by realjackwang** [reply]
Thanks
